x-pack/plugins/reporting/public/notifier/job_warning.tsx

```diff
@@ -28,18 +28,12 @@ export const getWarningToast = (
     { theme$: theme.theme$ }
   ),
   text: toMountPoint(
-    <Fragment>
-      <p>
-        <FormattedMessage
-          id="xpack.reporting.publicNotifier.warning.description"
-          defaultMessage="The report completed with issues."
-        />
-      </p>
+    <>
```

Comment:
Shouldn't there still be some warning text shown in this notifier?

Reply:
I'll defer to @gchaps for further explanation, but I do agree with her assessment that the description text does not really add new information beyond the title, so it seems cleaner to simplify this notification.

We could attempt to add more information about the reason for the warning, but IMO sending users to the Reporting UI is OK.

Reply:
(added a screenshot to the PR description too)

Reply:
I suggested removing the text because it repeats the title.  I think the message needs to say why you are sending the user to Stack Management > Reporting:

**CSV completed with issues**
Download the report now, or go to Stack Management > Reporting for details.

OR

**CSV completed with issues**
Download the report now, or go to Stack Management > Reporting for more on the issues.

Reply:
Thanks, I missed that the same message is in the title :+1: